Lib/annotationlib.py

```diff
@@ -77,6 +77,8 @@ def __init__(
         self.__forward_is_argument__ = is_argument
         self.__forward_is_class__ = is_class
         self.__forward_module__ = module
+        # These are always set to None here but may be non-None if a ForwardRef
+        # is created through __class__ assignment on a _Stringifier object.
```

Comment:
maybe move the `self.__owner__` assignment at the bottom of the `__init__` method above this comment, since that one is _not_ set unconditionally to `None`?